pyaedt/modeler/Object3d.py

```diff
@@ -2695,6 +2695,34 @@ def subtract(self, tool_list, keep_originals=True):
         self._primitives.modeler.subtract(self.name, tool_list, keep_originals)
         return self
 
+    @pyaedt_function_handler()
+    def wrap_sheet(self, object_name, imprinted=True):
+        """Execute the sheet wrapping around an object. This object can be either the sheet or the object.
+        If wrapping produces an unclassified operation it will be reverted.
+
+        Parameters
+        ----------
+        object_name : str, :class:`pyaedt.modeler.Object3d.Object3d`
+            Object name or solid object or sheet name.
+        imprinted : bool, optional
+            Either if imprint or not over the sheet. Default is `False`.
```

Comment:
@maxcapodi78 In the method signature, you set it to `True`. Maybe it should be `False` in the signature.
```suggestion
            Either if imprint or not over the sheet. Default is ``True``.
```